fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -131,6 +143,70 @@ fun OnboardingPage(
     }
 }
 
+@Composable
+private fun DescriptionText(
+    description: String,
+    linkifiedText: LinkifiedText?,
+) {
+    if (linkifiedText != null) {
+        LinkifiedText(
+            text = description,
+            linkifiedText = linkifiedText,
+        )
+    } else {
+        Text(
+            text = description,
+            color = FirefoxTheme.colors.textSecondary,
+            textAlign = TextAlign.Center,
+            style = FirefoxTheme.typography.body2,
+        )
+    }
+}
+
+/**
+ * A composable for displaying text that has a clickable linkified part.
+ * @param text The complete text.
+ * @param linkifiedText The linkified part of the text.
+ */
+@Composable
+private fun LinkifiedText(
```

Comment:
I wonder if having the Model and Func of the same name could be a little ambiguous?